### DISCUSSION TITLE
Draw Traces: pandapower.plotting.plotly.create_line_trace, legend question

### DISCUSSION THREAD

**Discussion by portal-cat** [e2nIEE/pandapower#1498] [Q&A] [answered]
When I use create_line_trace to create lines, many lines will be displayed in the legend of the final drawing (after draw_traces). Can we hide the lines in legend? (such as "showlegend=False" similar to the plotly) Or use "legendgroup" to show only one line in legend?

And also for the create_bus_trace, can we hide some buses in legend?  Thank you!

**Comment by jkisse** [comment, marked as answer]
related to #1506

**Reply by jkisse** [reply]
This should be solved by PR https://github.com/e2nIEE/pandapower/pull/1511
However, create_bus_trace was not modified. Could you please specify the desired behaviour? We also welcome pull requests if you can propose an implementation
